content/sensu-go/6.8/operations/maintain-sensu/disaster-recovery.md

```diff
@@ -0,0 +1,436 @@
+
+1. Stop each backend using the system manager:
+
+   {{< code shell >}}
+systemctl stop sensu-backend
+{{< /code >}}
+
+2. Delete the existing etcd directories for each backend:
+
+   {{< code shell >}}
+rm -rf /var/lib/sensu/sensu-backend/etcd/
+{{< /code >}}
+
+3. Start a new Sensu backend or cluster:
+
+   {{% notice warning %}}
+**IMPORTANT**: Update the example values in these commands according to your Sensu configuration before running the commands.
+{{% /notice %}}
+
+   {{< language-toggle >}}
+
+{{< code shell "Single backend startup" >}}
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380 \
+--etcd-initial-cluster-token sensu-backend-01 \
+--etcd-initial-advertise-peer-urls http://localhost:2380 \
+--etcd-advertise-client-urls http://localhost:2379
+{{< /code >}}
+
+{{< code shell "Clustered backend startup" >}}
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--etcd-initial-cluster-token sensu-backend-01 \
+--etcd-initial-advertise-peer-urls http://sensu-backend-01:2380 \
+--etcd-advertise-client-urls http://sensu-backend-01:2379
+
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--etcd-initial-cluster-token sensu-backend-02 \
+--etcd-initial-advertise-peer-urls http://sensu-backend-02:2380 \
+--etcd-advertise-client-urls http://sensu-backend-02:2379
+
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--etcd-initial-cluster-token sensu-backend-03 \
+--etcd-initial-advertise-peer-urls http://sensu-backend-03:2380 \
+--etcd-advertise-client-urls http://sensu-backend-03:2379
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+4. Confirm that the new Sensu backend or cluster is running:
+
+   {{< code shell >}}
+systemctl status sensu-backend
+{{< /code >}}
+
+   The response should indicate `active (running)`.
+
+5. Copy the etcd snapshot file to each cluster member so that all members will be restored using the same snapshot.
+
+6. Restore the Sensu configuration from the etcd snapshot to the running backend or cluster:
+
+   {{% notice warning %}}
+**IMPORTANT**: Update the example values in these commands according to your Sensu configuration before running the commands.
+{{% /notice %}}
+
+   {{< language-toggle >}}
+
+{{< code shell "Restore a single backend" >}}
+ETCDCTL_API=3 etcdctl snapshot restore sensu_etcd_snapshot.db \
+--name sensu-backend-01 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
```

Comment:
```suggestion
--initial-cluster sensu-backend-01=http://sensu-backend-01:2380 \
```

If this is for a single backend, then it doesn't make sense to have 2 other backends as part of the restore command.